packages/earth-map/src/components/last-viewed-place/last-viewed-place.tsx

```diff
@@ -24,9 +24,10 @@ import { useTranslation } from 'react-i18next';
 
 interface IProps {
   place: ILastViewedPlace;
+  group: any;
```

Comment:
We'd need to refactor this later to `groups` throughout the app. Might be confusing for others.